manager/src/main/java/com/redhat/service/smartevents/manager/BridgesServiceImpl.java

```diff
@@ -111,6 +120,10 @@ public Bridge createBridge(String customerId, String organisationId, String owne
             throw new AlreadyExistingItemException(String.format("Bridge with name '%s' already exists for customer with id '%s'", bridgeRequest.getName(), customerId));
         }
 
+        // Create resource on AMS - raise an exception is the organisation is out of quota
+        CreateResourceRequest createResourceRequest = craftAMSCreateResourceRequest(organisationId);
+        String subscriptionId = accountManagementService.createResource(createResourceRequest).await().atMost(Duration.ofSeconds(2)).getSubscriptionId();
```

Comment:
Are two seconds enough here?

Reply:
This is a synchronous call that we have to perform while we process the user's request. I would set it at 5 seconds max as it is important not to hang too much in case AMS does not respond. I'll catch the exception as well :+1: